src/test/java/com/celements/model/classes/fields/DateFieldTest.java

```diff
@@ -22,8 +22,8 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
```

Comment:
please do not override the default setUp method. Instead add a different named method with Before annotation.